nautobot/dcim/migrations/0030_migrate_region_and_site_data_to_locations.py

```diff
@@ -151,6 +163,19 @@ def create_site_location_type_locations(
 
     # Set existing top level locations to have site locations as their parents
     site_lt_locations = location_class.objects.filter(location_type=site_lt)
+    for site in site_class.objects.all():
+        for location in site_lt_locations:
+            # move tags from Site to Location
+            for tagged_item in tagged_item_class.objects.filter(content_type=site_ct, object_id=site.pk):
```

Comment:
Triple-nested for loop here feels potentially very slow, especially given that we're doing at least two queryset evaluations in each inner loop. Can we perhaps instead pair sites-to-corresponding-locations in advance and iterate over both in a single loop?